mappings/net/minecraft/client/gui/screen/Screen.mapping

```diff
@@ -8,12 +8,16 @@ CLASS net/minecraft/unmapped/C_wrmtlwqx net/minecraft/client/gui/screen/Screen
 	FIELD f_htifmqlv title Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_kmcqeiie width I
 	FIELD f_kreiluct children Ljava/util/List;
+	FIELD f_lpsgbjcj EXTRA_SPACE_AFTER_FIRST_TOOLTIP_LINE I
```

Comment:
Maybe take extra out?

Reply:
I agree this is quite long and would probably be fine without extra